metagraph/src/graph/representation/succinct/boss_chunk_construct.cpp

```diff
@@ -33,7 +33,7 @@ using utils::get_first;
 using utils::get_first_type_t;
 using TAlphabet = KmerExtractorBOSS::TAlphabet;
 
-constexpr size_t ENCODER_BUFFER_SIZE = 100'000;
+const size_t ENCODER_BUFFER_SIZE = 100'000;
```

Comment:
It's just a usual global const variable, and we use `const ...` for them.

Actually, I believe the compiler comprehends them as constexpr anyway. Besides, this parameter doesn't have to be constexpr, since the compilation doesn't depend on it.